GameModel.md

```diff
@@ -0,0 +1,78 @@
+The game is as follows
+
+There are 3 initial parties: Client (C), Worker (W), Verifier (V)
+
+The Client comes with a take-it-or-leave-it offer that Worker and Verifier can agree to sign. The offer is a contract to do calculations (Work) for a Payment, which is split in a predetermined way between W and V. The Worker shall calculate all the Work, while the Verifier shall calculate only a small portion of Work that is called Sample (described more fully at Stage 2). 
+
+According to the contract, the Payment, the Worker Collateral Stake (w) and the Verifier Collateral Stake (v) are deposited to Escrow, to be released under certain conditions described in this text. Since calculations incure costs (hardware, electricity), the incentive to report false results without doing actual calculations is countered by providing collateral stakes that can be slashed if cheating is detected. It is economically logical that v is larger or equal to w, where the case w=v has certain merits of symmetry against attacks on stake. It is also economically logical that Verifier's work should be more expensive (per unit of calculations). This is because Verifier's stake is locked for approximately the same time as Worker, but Verifier's total reward is less than Worker's total reward (since much less work is done).
+
+The first taker of the contract will become Worker. However, the Verifier should be randomly chosen between a sufficiently large pool of volunteers who have made an Initial Deposit (not necessarily equal to v, can be smaller), so as to reduce the possibility of collusion between W, V or C, V.
```

Comment:
As we discussed, the new idea is that Client is free to choose both Worker and Verifier and this part is outside of the protocol. The protocol starts when the choice is already made.

So if there will be a non-random Verifier (chosen by the Client), would it affect economic incentives of the parties?

Reply:
Sure it would affect the incentives in many different ways. It would take away the safeguards that come from the fact that parties were not initially related to the Client and therefore not colluding with the Client by default, and that such a collusion was relatively rare. If such a collusion is trivial, then the main remaining safeguard would be the Arbitrage. It is not an impossible setting, but it lays much greater responsibility on the Arbitrage and gives more incentives for Arbiters corruption.

As a side note, it may be hard for the honest Client to pick an honest Verifier, since the Client is not well versed in random procedures and is likely to choose whoever comes first, or from a small initial pool, and those could more easily be Worker's collaborators.